### ISSUE TITLE
Bind OctoPrint on 0.0.0.0:5001

### ISSUE THREAD

**Issue by leigh-johnson** [bitsy-ai/printnanny-os#191]
## **Describe the bug**
OctoPrint currently binds to 127.0.0.1:5001, with nginx proxy passing 0.0.0.0:80/octoprint/ to the OctoPrint service.

However, some external apps (like Cura and Prusa Slicer) expect OctoPrint to be available on 0.0.0.0:5001 

## Issue.txt
```
-----------------------
Build Info:  |
-----------------------
IMAGE_NAME = printnanny-release-image-raspberrypi4-64-20230106203240
DISTRO_PRETTY = PrintNanny Linux 0.5.1 (Emerald)
VARIANT_NAME = PrintNanny OS Core
VARIANT_ID = core
DISTRO = printnanny
DISTRO_NAME = PrintNanny Linux
DISTRO_VERSION = 0.5.1
DISTRO_CODENAME = Emerald
ID_LIKE = BitsyLinux
HOME_URL = https://printnanny.ai
BUG_REPORT_URL = https://github.com/bitsy-ai/printnanny-os/issues
YOCTO_VERSION = 4.0.2
YOCTO_CODENAME = Langdale
MACHINE = raspberrypi4-64
TUNE_PKGARCH = cortexa72
-----------------------
Layer Revisions:      |
-----------------------
meta              = langdale:a3e3b740e140d036122f7b11e2ac452bda548444
meta-poky         = langdale:a3e3b740e140d036122f7b11e2ac452bda548444
meta-yocto-bsp    = langdale:a3e3b740e140d036122f7b11e2ac452bda548444
meta-raspberrypi  = libcamera-v0.0.3:104a8b9290ab4da92c8f693e2fdc42a6396975d2
meta-oe           = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-python       = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-multimedia   = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-bitsy        = main:957dfaa51204ce7739a3d222e50c5c33d656481e
meta-neural-network = master:fed68f0a659c9b6431985c156141dc28dc4c2275
meta-printnanny   = main:957dfaa51204ce7739a3d222e50c5c33d656481e
meta-networking   = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-filesystems  = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-initramfs    = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-webserver    = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-swupdate     = master:5e611f44db1b9333342ed61d77a3c3ed7be69c2b
workspace         = langdale:a3e3b740e140d036122f7b11e2ac452bda548444
meta-microcontroller = master:9f396e595b2d639be7f2b494f4a61e48d9495058
```

**Comment by leigh-johnson** [comment]
🎉 Fixed! I connected Cura -> OctoPrint in the latest nightly build. Closing. This fix will be available in v0.5.2

<img width="892" alt="Screenshot 2023-01-08 at 3 48 38 PM" src="https://user-images.githubusercontent.com/2601819/211225099-6c7d1bcf-da9c-42aa-adfe-0d04ca71ced6.png">

**Comment by leigh-johnson** [comment]
I found a better way to do this, since :5001 connects directly to OctoPrint's web server (by-passing nginx). 

The discovery plugin is responsible for ZeroConf broadcast, and I can configure discoverable port/path. I'm going to revert binding Octoprint's web server to 0.0.0.0 and continue using nginx to proxy connections
https://docs.octoprint.org/en/master/bundledplugins/discovery.html#
